### ISSUE TITLE
How to train on LIVE Challenge to get SRCC around 0.85?

### ISSUE THREAD

**Issue by witwitchayakarn** [zwx8981/DBCNN-PyTorch#7]
Hi, me again.
Can you suggest/give more detail how to train on LIVE Challenge to get SRCC around 0.85?

I've trained on LIVE release2 using your code and got SRCC very close to 0.968 but on LIVE Challenge, I could get just around 0.81. I've tried changing learning rate, number of epochs, and using SGD instead of Adam on both FC train and FULL train. Did I miss anything or anything I should try?

Thank you in advance, really appreciate it.

**Comment by zwx8981** [comment]
The difference between original MatConvNet implementation and this PyTorch re-implementation along with the randomness brought by training-testing set split may indeed cause the results fluctuating. Even though, I think the average srcc on LIVE Challenge should be around 0.84~0.85, did you train-test using default hyper-parameters?

**Comment by witwitchayakarn** [comment]
I'm so sorry. I've just run your code directly and now I got SRCC around 0.85 on LIVE Challenge. Previously, I copied your code and run on a notebook. I must've missed something. Sorry again and thank you very much for your help.
